api/src/main/java/com/exelaration/abstractmemery/services/implementations/MemeMetadataServiceImpl.java

```diff
@@ -27,4 +33,17 @@ public Meme save(Meme meme) {
   public void deleteById(Integer id) {
     memeRepository.deleteById(id);
   }
+
+  public ArrayList<String> getMemes() {
+    ArrayList<String> memeNames = memeRepository.getMemes();
+    ArrayList<String> memeURLs = new ArrayList<String>();
+    String directory = "/app/src/main/resources/memes/";
```

Comment:
@mchughmk  I know that this is what you recommended this morning, but I think I'm still a little confused on why the file encoding retrieval is in MemeMetadataService. I thought this service was solely to retrieve database information. If so, wouldn't having it grab the encoded file violate the single responsibility part of the service?

Reply:
Should the memeService be the one that makes all the calls to the other services and sends off the list of memeURLs?